store/tikv/oracle/oracle.go

```diff
@@ -142,6 +148,25 @@ func GoTimeToTS(t time.Time) uint64 {
 	return uint64(ts)
 }
 
+// CompareTS is used to compare two timestamps.
+// If tsoOne > tsoTwo, returns 1.
+// If tsoOne = tsoTwo, returns 0.
+// If tsoOne < tsoTwo, returns -1.
+func CompareTS(tsoOne, tsoTwo uint64) int {
```

Comment:
What's the difference with just comparing the int value of `tsoOne` and `tsoTwo`?

Reply:
The logic is migrated from PD, I think there is no difference.